proc/proc.go

```diff
@@ -0,0 +1,32 @@
+package proc
+
+import (
+	"github.com/scootdev/scoot/runner"
+)
+
+type JobType int
```

Comment:
Where is this file used vs the thrift def, i.e. what is the boundary.

It's a bit confusing to have jobtype as string in some places and int here.